forge-gui/res/cardsfolder/upcoming/invasion_of_innistrad_deluge_of_the_dead.txt

```diff
@@ -0,0 +1,22 @@
+Name:Invasion of Innistrad
+ManaCost:2 B B
+Types:Battle Siege
+Defense:4
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ When CARDNAME enters the battlefield, target creature an opponent controls gets -13/-13 until end of turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select target creature an opponent controls | NumAtt$ -13 | NumDef$ -13
+DeckHas:Ability$Counters|Token|Graveyard & Type$Zombie
```

Comment:
My preference would be to split AI hints as appropriate on each face... this method is more confusing to review and for future us to copy/paste effectively